packages/design-system/src/components/Promo/Promo.js

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import { Button } from '../Button';
+
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import styles from './style.scss';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'promo';
+
+export const Promo = props => {
+  const {
+    buttonText,
+    children,
+    className,
+    header,
+    image,
+    light,
+    linkText,
+    onButtonClick,
+    onClose,
+    onLinkClick,
+    size,
+    img,
+    ...restProps
+  } = props;
+
+  const mergedWrapperClassNames = getMergedClassNames(
+    cx({
+      [`${baseClass}--light`]: light,
+      [`${baseClass}--${size}`]: size,
+      [`${baseClass}`]: true,
+    }),
+    className
+  );
+  const shouldRenderFooter = buttonText || linkText;
+
+  return (
+    <div className={mergedWrapperClassNames} {...restProps}>
```

Comment:
is there no way to use the previously developed Banner component?

Reply:
There are some differences between them:

- icons / image alignement
- business case: promo / alert
I would avoid situation when changes in banner will affect banner component, because these components will be used in different cases.